cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -145,8 +146,10 @@ func (cmd *command) Run() error {
 	cmd.duration = time.Since(start)
 
 	// import certificate
-	if err := cmd.importCertificate(); err != nil {
-		return err
+	if !cmd.opts.SkipCertImport {
```

Comment:
Why we use a new flag for that? For me that is the regular CI or non-interactive scenario, so please re-use the existing flags we have.